### DISCUSSION TITLE
How to commit a grammar upstream?

### DISCUSSION THREAD

**Discussion by GordianDziwis** [tree-sitter/tree-sitter#807] [Grammars]
I am working on a sparql grammar and the grammar is complete, how can I make this grammar official? 

https://github.com/BonaBeavis/tree-sitter-sparql

**Comment by jimhester** [comment]
Similarly I have written an R tree-sitter grammar (https://github.com/jimhester/tree-sitter-r) and had the same question.

**Reply by alemuller** [reply]
The website home is at [docs/index.md](https://github.com/tree-sitter/tree-sitter/blob/master/docs/index.md). You can edit and send a PR.
